docs/src/site/twiki/BacklogMetricEmitterService.twiki

```diff
@@ -0,0 +1,51 @@
+---+Back Log Emitter Service
+
+Back Log Emitter Service periodically calculates the process backlog and send data to graphite.
+
+The emission and calculation of backlog are controlled by the below properties in startup.properties :
+
+<verbatim>
+*.falcon.backlog.metricservice.emit.interval.millisecs=60000
+*.falcon.backlog.metricservice.recheck.interval.millisecs=60000
+</verbatim>
+
+---+How Backlog is Calculated
+
+Back Log Emitter Service is a listener to :
+
+   1. EntitySLAAlertService
+   1. !WorkflowExecutionListener
+
+
+EntitySLAListner service notifies !BacklogEmitter whenever a feed misses highSla or process misses its shouldEndIn sla.
+
+!WorkflowExecutionListener notifies !BacklogEmitter service whenever a process get completed.
+
+Back Log Emitter service keep track of all the process which have missed an SLA for a particular time and calculates backlog.Once a notification comes that process has completed it updates the backlog.This service keeps on sending data to Graphite for each process which is missing SLA.
+
+*Back Log Calculation:*
```

Comment:
It will be nice to provide the definition of backlog at the top of the document, along with this example.